src/components/Organizations/style.js

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components'
+
+export const OrganizationsDiv = styled.div`
+  width: 100%;
+
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 30%);
+  align-items: center;
+  justify-content: center;
+  gap: 3rem 5rem;
+  flex-wrap: wrap;
+
+  margin-top: 3.5rem;
+
+  @media screen and (max-width: 1366px) {
+    grid-template-columns: repeat(auto-fill, 28%);
+  }
+
+  @media screen and (max-width: 1024px) {
```

Comment:
Como acontece a mudança de template da tela em telas um pouco menores acho que seria interessante deixar as duas colunas para telas um pouco menores com 925px
```suggestion
  @media screen and (max-width: 925px) {
```